addons/l10n_eu_service/wizard/wizard.py

```diff
@@ -0,0 +1,152 @@
+        return eu_country_group.country_ids - self.compute_default_value_todo_country() - user.company_id.country_id
+
+    def compute_default_value_todo_country(self):
+        user = self.env.user
+        eu_country_group = self.env['res.country.group'].search([('name', '=', 'Europe')])
+        eu_fiscal = self.env['account.fiscal.position'].search(
+            [('country_id', 'in', eu_country_group.country_ids.ids),
+             ('vat_required', '=', False), ('auto_apply', '=', True)])
+        return eu_country_group.country_ids - eu_fiscal.mapped('country_id') - user.company_id.country_id
+    
+    chart_id = fields.Many2one(
+        "account.account", string="Chart of Accounts", required=True,
+        default=compute_default_value_chart)
+    company_id = fields.Many2one(
+        'res.company', string='Company', required=True, 
+        related='chart_id.company_id', readonly=True)
+    fiscal_position_id = fields.Many2one(
+        'account.fiscal.position', string='Fiscal Position',
+        help="Optional fiscal position to use as template for general account mapping. "
+             "Should usually be your current Intra-EU B2C fiscal position. "
+             "If not set, no general account mapping will be configured for EU fiscal positions.",
+        default=compute_default_value_fiscal_position)
+    tax_id = fields.Many2one(
+        'account.tax', string='Service VAT', default=compute_default_value_tax, required=True,
+        help="Select your current VAT tax for services. This is the tax that will be mapped "
+             "to the corresponding VAT tax in each EU country selected below.")
+    account_collected_id = fields.Many2one(
+        "account.account", string="Tax Collection Account",
+        help="Optional account to use for collecting tax amounts when selling services in "
+             "each EU country selected below. If not set, the current collecting account of "
+             "you Service VAT will be used.")
+    done_country_ids = fields.Many2many(
+        'res.country', 'l10n_eu_service_country_rel_done',
+        string='Countries already setup', default=compute_default_value_done_country)
+    todo_country_ids = fields.Many2many(
+        'res.country', 'l10n_eu_service_country_rel_todo',
+        string='Countries to setup', default=compute_default_value_todo_country, required=True)
+
+    @api.multi
+    def compute_eu_service(self):
+        info = self[0]
+        imd = self.env['ir.model.data']
+        tax_code = self.env['account.tax.code']
+        tax_rate = self.env["l10n_eu_service.service_tax_rate"]
+        account_tax = self.env['account.tax']
+        fpos = self.env['account.fiscal.position']
+        chart_xid = 'l10n_eu_service.tax_chart_service_eu_company_%s' % info.company_id.name
+        chart_id = imd.xmlid_to_res_id(chart_xid)
+        if not chart_id:
+            vals = {
+                'name': 'EU VAT Services Taxes',
```

Comment:
We should rename this to something like `_("EU MOSS VAT Chart") + " - %(company)s"` to make it translatable.